senders/victorops/api/alert.go

```diff
@@ -0,0 +1,63 @@
+package api
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+// CreateAlertRequest the API request to be made to
+// create a victorops alert
+type CreateAlertRequest struct {
+	MessageType       MessageType `json:"message_type,omitempty"`
+	EntityID          string      `json:"entity_id,omitempty"`
+	EntityDisplayName string      `json:"entity_display_name,omitempty"`
+	StateMessage      string      `json:"state_message,omitempty"`
+	StateStartTime    int64       `json:"state_start_time,omitempty"`
+}
+
+// MessageType is the type of a victorops alert
+type MessageType string
+
+// Various possible MessageTypes
+const (
+	Critical        MessageType = "CRITICAL"
+	Warning         MessageType = "WARNING"
+	Acknowledgement MessageType = "ACKNOWLEDGEMENT"
+	Info            MessageType = "INFO"
+	Recovery        MessageType = "RECOVERY"
+)
+
+// CreateAlert creates a new alert in the victorops timeline
+func (client *Client) CreateAlert(routingKey string, alert CreateAlertRequest) error {
+	if alert.MessageType == "" {
+		return fmt.Errorf("field MessageType cannot be empty")
+	}
+
+	body := new(bytes.Buffer)
+	err := json.NewEncoder(body).Encode(alert)
```

Comment:
Let's use json.Marshal, as elsewhere in Moira, [example](https://github.com/moira-alert/moira/blob/master/metric_source/remote/response.go#L28) (this is the reverse example, which is json.Unmarshal response body)